src/main/scala/devices/debug/SBA.scala

```diff
@@ -138,14 +139,14 @@ object SystemBusAccessModule
     val hasSBData2And3 = (cfg.maxSupportedSBAccess == 128)
     val hasData        = Seq(true, hasSBData1, hasSBData2And3, hasSBData2And3)
 
-    val SBDATAFieldsReg = Seq.fill(4)(Seq.fill(4)(Reg (UInt(8.W))))
+    val SBDATAFieldsReg = RegInit(VecInit(Seq.fill(4) {VecInit(Seq.fill(4) {0.U(8.W)} )} ))
```

Comment:
same here

Reply:
So are we Ok with this being reset by the module reset? This is a change in functionality.

Reply:
You're right, this did change functionality.  I changed to a different method that should maintain existing function.